microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -552,6 +637,54 @@ The functions tag is not unique to your installation of Microsoft Edge. The tag
 If Microsoft Edge detects SSL connection timeouts, certificate errors, or other network issues that may be caused by a captive portal, it sends a request to `http://edge.microsoft.com/captiveportal/generate_204` and checks the response code. Captive portals include a Wi-Fi network at a hotel or airport. If the request is redirected to another URL, Microsoft Edge opens the URL in a new tab, assuming that it is a sign-in page.  Requests to the captive portal detection page are a stateless service. Requests aren't logged, and cookies aren't sent or saved.  On Windows platforms, Microsoft Edge uses a Windows captive portal service. Otherwise, the Microsoft Edge captive portal service is used. You can turn off the service. Go to `edge://settings/privacy`, and turn off the **Use a web service to help resolve navigation errors** setting.
 
 
+<!-- ====================================================================== -->
+## Rewards
+
+Microsoft Rewards (the "Program") enables you to earn redeemable points for activities such as qualified searches, acquisitions, and other offers from Microsoft. These points can then be redeemed for gift cards or non-profit donations or automatic contributions to subscriptions. Microsoft Edge has built-in support for all existing avenues for earning points, and also introduces new and exclusive means to earn more points. You can track your points, and you can stay up-to-date with limited period offers to get the most out of the program.
+
+For non-members, Microsoft Edge presents personalized joining offers based on the page you're on. Once you engage and choose to enroll, Microsoft Edge uses your Microsoft account identity to sign you up and start the rewards service.  The rewards service automatically adds points to your account for searching, playing, and shopping on Microsoft Edge. Microsoft Edge doesn't send your personal information or browsing data to any third-party services.
+
+Microsoft Edge periodically offers custom browsing streak offers, for browsing on Microsoft Edge for a few minutes every day for a limited period. You can track your streak progress; points earned and discover new offers on the [Rewards dashboard](https://rewards.bing.com/) or in the **Action center** in the Microsoft Edge sidebar.
```

Comment:
```suggestion
Microsoft Edge periodically offers custom browsing streak offers for a limited period, for browsing on Microsoft Edge for a few minutes every day. You can track your streak progress, track your points earned, and discover new offers on the [Rewards dashboard](https://rewards.bing.com/) or in the **Action center** in the Microsoft Edge sidebar.
```
* Use verb parallelism in mid-sentence list.  
* Move "for a limited period" where it fits best.  Tech review this change @majenny